src/Components/Web.JS/src/Platform/Circuits/CircuitManager.ts

```diff
@@ -297,6 +307,10 @@ export class CircuitManager implements DotNet.DotNetCallDispatcher {
     this.disconnect();
   }
 
+  private setCircuitStatus(status: 'connecting' | 'connected' | 'disconnected') {
```

Comment:
Also the current possible values are `'connecting'`, `'connected'`, and `'disconnected'`. In the period between reconnect attempts, the status is `'disconnected'`. Also, the status is `'connecting'` when the circuit is performing its initial connection.

I think this behavior accurately reflects the true state of the circuit, but I wonder if it's more useful to make the status `''` as long as a disconnection has never happened, `'reconnecting'` when the circuit is disconnected but the reconnect UI is visible, and `disconnected` after reconnection has failed. Thoughts on this?

Reply:
Actually, it's not super clear to me what the expected usage pattern would be for this new custom CSS property. It seems that it would primarily be useful when using [container style queries](https://developer.mozilla.org/docs/Web/CSS/@container#container_style_queries), which not yet supported in all major browsers.

We could alternatively put an attribute on the `<html>` element like `data-circuit-status` and then developers could use an [attribute selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) to customize styles depending on what that attribute's value is.

I'm probably going to scope this out of the PR for now, and we can think about adding an attribute or custom CSS property at a later time.